server/sandbox_stop.go

```diff
@@ -16,7 +16,13 @@ func (s *Server) StopPodSandbox(ctx context.Context, req *pb.StopPodSandboxReque
 	logrus.Debugf("StopPodSandboxRequest %+v", req)
 	sb, err := s.getPodSandboxFromRequest(req.PodSandboxId)
 	if err != nil {
-		return nil, err
+		if err == errSandboxIDEmpty {
+			return nil, err
+		}
+
+		resp := &pb.StopPodSandboxResponse{}
+		logrus.Warnf("could not get sandbox %s, it's probably been stopped already: %v", req.PodSandboxId, err)
+		return resp, nil
```

Comment:
Yeah, let us add comments so it is clear that we are trying to adhere to kubelet's current expectations.